pkg/cmd/container/rename.go

```diff
@@ -44,7 +44,7 @@ func Rename(ctx context.Context, client *containerd.Client, containerID, newCont
 	if err != nil {
 		return err
 	}
-	hostst, err := hostsstore.NewStore(dataStore)
+	hostst, err := hostsstore.New(dataStore, options.GOptions.Namespace)
```

Comment:
HostsStore signature got changed to be per-namespace - align with other stores, and guarantees better isolation between namespaces and less "global" locking.